source/_drafts/Video-js-6-7-1-released.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Video.js 6.7.1 released
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+---
+Video.js 6.7.1 was released this week. This comes a month and a half since the first [6.6.0 release][6.6.0] (which we forgot to blog about) and only a week since the latest patch release ([6.6.3][], which is now latest).
+
+6.7 has two awesome new features and a couple helpers:
+- a working `playerresize` event
+- a new mediator type for middleware
+- `getPlayer` and `getAllPlayers` helper on `videojs`.
+
+## Netlify
+
+Recently, we also switched over all our online properties to run on [Netlify][]. This is great because it gives us HTTPS via Let's Encrypt but also allows us much better automation for the website, docs website, and blog. Since the docs website is tied to the main Video.js repo and has a build per PR, we also had Netlify generate an example page for the PR based on the sandbox examples. [Here's the example page][example] for a [recent PR][].
+
+## `playerresize`
+
+This new event will fire each time the player is resized. It will fire when going fullscreen and when exiting fullscreen and when resizing the player via the dimension methods or if the player is in fluid mode and the window is resized.
+It uses the new [ResizeObserver][] in Chrome 64 and where-ever it is available. If it isn't available, a polyfill can be passed in or it will use its fallback. The fallback uses an absolutely positioned, hidden iframe that's the size of the player and then retriggers the iframe's `resize` event on a debounced handler.
+Also, I wanted to note that the `resize` player event does not refer to the size of the player itself but rather to the native [`resize` event][resize], which is triggered when the videoHeight or videoWidth has changed.
+
+## Mediator type for middleware
+
+This is the first major feature for middleware since they were released in 6.0. The  main reason for the mediator is for middleware to be able to cancel requests to `play()`. Thus, they are currently limited to `play()` and `pause()` calls currently. Once we iron out the details we hope to enable it for further functionality.
```

Comment:
Minor pet peeve, but middleware is not a plural noun.

Reply:
middleware is an uncountable noun and doesn't really have a plural form.